pulser-pasqal/pulser_pasqal/backends.py

```diff
@@ -223,20 +223,55 @@ def __init__(
             config=config,
             mimic_qpu=mimic_qpu,
         )
-        # To be deleted once this PR is released: https://github.com/pasqal-io/Pulser/pull/890
-        self._config = type(self.default_config)(
-            **{
-                **self.default_config._backend_options,
-                **(config._backend_options if config else {}),
-            }
-        )
+
+    def run(
+        self, job_params: list[JobParams] | None = None, wait: bool = False
+    ) -> RemoteResults:
+        """Runs the sequence on this remote emulator and returns the result.
+
+        Args:
+            job_params: A list of parameters for each job to execute. If the
+                sequence is parametrized, the values for all the variables
+                necessary to build the sequence must be given for
+                each job, under the 'variables' field. If not given, a single
+                job is executed.
+            wait: Whether to wait until the results of the jobs become
+                available.  If set to False, the call is non-blocking and the
+                obtained results' status can be checked using their `status`
+                property.
+
+        Warning:
+            Unlike a 'QPUBackend', this backend does not expect a value for
+            "runs" in each entry of 'job_params'. If provided, this value is
+            ignored. If you wish to set the total number of bitstring counts
```

Comment:
This is a slight change with the current bahaviour. Currently, the `runs` value is used if the user doesn't specify the observables in his config. 

This is a good step towards our goal to get rid of job params but just wanted to make sure this was intentional